backend-rust/src/indexer.rs

```diff
@@ -509,12 +509,6 @@ LIMIT 1
             last_cumulative_num_txs: last_block.cumulative_num_txs,
         };
 
-        let blocks_processed = Counter::default();
-        registry.register(
```

Comment:
I think this is still interesting to have

Reply:
Exactly the same as:

indexer_processor_batch_size_sum